Dockerfile

```diff
@@ -5,6 +5,11 @@ FROM python:3.6
 # to the terminal with out buffering it first
 ENV PYTHONUNBUFFERED 1
 
+# setup nginx
```

Comment:
I think we don't necessarily need to have nginx setup within the container. This will be setup on the host that the container will be running on - it'll manage traffic coming into the host, and send it to the container.